SitumWayfinding/Classes/WayfindingLibrary/Protocols/OnMapReadyListener.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  OnMapReadyListener.swift
+//  SitumWayfinding
+//
+//  Created by Lapisoft on 18/1/22.
+//
+
+import Foundation
+
+/**
+ Delegate that get notified when the map is ready to interact and fully loaded.
+ */
+public protocol OnMapReadyListener {
+    /**
+     Method that notifies when the map is fully loaded. This will be called after call load() on library when all
+     the resources are loaded, and the user can start to interact with the map.
+     */
```

Comment:
Que vós parece a seguinte descripción?
Method that notifies that it is safe to perform operations over the map. After the SitumMapsLibrary load() method is call, the module has to load the map on screen and obtain cartographic information. During this time the module is not properly initialized and operations over the map are not guaranteed to end as expected.

Reply:
Resolto en  https://github.com/situmtech/ios-swift-library-wayfinding/pull/57